testing/endpoint_v2.go

```diff
@@ -43,22 +44,26 @@ func (endpoint *Endpoint) MsgSendPacketWithSender(timeoutTimestamp uint64, paylo
 		return channeltypesv2.Packet{}, err
 	}
 
-	// TODO: parse the packet from events instead of from the response. https://github.com/cosmos/ibc-go/issues/7459
-	// get sequence from msg response
-	var msgData sdk.TxMsgData
-	err = proto.Unmarshal(res.Data, &msgData)
-	if err != nil {
-		return channeltypesv2.Packet{}, err
-	}
-	msgResponse := msgData.MsgResponses[0]
-	var sendResponse channeltypesv2.MsgSendPacketResponse
-	err = proto.Unmarshal(msgResponse.Value, &sendResponse)
-	if err != nil {
-		return channeltypesv2.Packet{}, err
```

Comment:
This needs to be implemented as a function as is specified in the original issue: https://github.com/cosmos/ibc-go/issues/7856